lib/template-loader.php

```diff
@@ -155,11 +155,11 @@ function gutenberg_find_template( $template_file ) {
 			'post_name__in'  => $slugs,
 			'orderby'        => 'post_name__in',
 			'posts_per_page' => 1,
+			'no_found_rows'  => true,
 		)
 	);
 
-	$template_posts        = $template_query->get_posts();
-	$current_template_post = array_shift( $template_posts );
+	$current_template_post = $template_query->next_post();
```

Comment:
```suggestion
		$current_template_post = $template_query->have_posts() ? $template_query->next_post() : null;
```

It looks like adding a check for `have_posts` fixes the issue. The other code in the file already checks if `$current_template_post` is empty, so null should work fine here